second_response_to_reviewers.tex

```diff
@@ -0,0 +1,366 @@
+approximation) and the actual result.
+
+After careful examination of how this section is written we do not see how it
+can be improved and feel it is clear. Specifically this sentence: `This
+demonstrates that assuming a given interaction between two
+IPD strategies can be summarised with a set of utilities as
+shown in (1) is not correct.'.
+
+\begin{verbatim}
+The authors should avoid jargon because it doesn’t help readability. For instance, in
+Section III(c) the authors state “this is explained by the need of strategies to have a
+steady hand when interacting with their own kind. Acting stochastically increases the
+chance of friendly fire.”. What is a “steady hand” and what is “friendly fire” in an
+IPD game?
+\end{verbatim}
+
+We have replaced this comments with more standard language.
+
+\begin{verbatim}
+Here are some of the questions I have after reading the
+paper. (The first two points deal with the realism of the models; the last two with exploiting
+the paper’s findings.)
+
+- Suppose 10 people decide to play an IPD game. The authors only allowed pairs of
+strategies in the population. If k individuals happen to pick the same strategy, what
+is the likelihood the other 10 − k people will pick the same alternative strategy?
+- The strategies evolved under a Moran process. Under a Moran process each round
+one player must change his strategy 1 . Is there an example of a real-world social
+dilemma or a human experiment where strategy changes resulted from a Moran pro-
+cess or something similar? (This issue is particularly important because the authors
+specifically trained some IPD strategies to excel at the Moran process.)
+\end{verbatim}
+
+We refer the reviewer to the many references included and the vast quantity of
+literature on the Moran process. This is not a new model but a well understood
+model of evolutionary processes with applications both to human and biological
+interactions. It seems like the review mainly questions the Moran process itself
+as opposed to our use of it.
+
+\begin{verbatim}
+Suppose the population is using either strategy A or strategy B. Mutation was
+not
+allowed in the authors’ simulation so no other strategies can be used. How
+useful then
+is it to know some other strategy C can successfully invade strategy B if
+strategy C
+isn’t available?
+\end{verbatim}
+
+This is more useful than the already published (and heavily cited) works in this
+area that look at the score of A against B as opposed to the ability of A to
+invade/defend B.
+
+\begin{verbatim}
+How useful is it knowing that strategy A can or cannot resist invasion by
+strategy
+B when players rarely know exactly what strategy their opponent is using? In
+par-
+ticular, before the game begins players won’t know what strategy others intend
+to
+use.
+\end{verbatim}
+
+This feels like a question about the entirety of the research field on the IPD.
+
+We refer the referee to the vast quantity of literature in this field and in
```

Comment:
Let's just say prior literature. It's clear that the reviewer is not an expert and we've made that point already. There's also another reference to "vast quantity" above that is a little over the top IMO. I completely understand the frustrations of course.

Reply:
Very wise (I had already removed some comments that stemmed from frustration but you're completely right to remove them all.) :+1: